lib/content-info/index.js

```diff
@@ -0,0 +1,384 @@
+    * parameter     = reg-parameter / ext-parameter
+    * ext-parameter = parmname "*" LWSP "=" LWSP ext-value
+    * parmname      = 1*attr-char
+    * ext-value     = charset  "'" [ language ] "'" value-chars
+                   ; like RFC 2231's <extended-initial-value>
+                   ; (see [RFC2231], Section 7)
+    * charset       = "UTF-8" / "ISO-8859-1" / mime-charset
+    * mime-charset  = 1*mime-charsetc
+    * mime-charsetc = ALPHA / DIGIT
+                   / "!" / "#" / "$" / "%" / "&"
+                   / "+" / "-" / "^" / "_" / "`"
+                   / "{" / "}" / "~"
+                   ; as <mime-charset> in Section 2.3 of [RFC2978]
+                   ; except that the single quote is not included
+                   ; SHOULD be registered in the IANA charset registry
+    * language      = <Language-Tag, defined in [RFC5646], Section 2.1>
+    * value-chars   = *( pct-encoded / attr-char )
+    * pct-encoded   = "%" HEXDIG HEXDIG
+                   ; see [RFC3986], Section 2.1
+    * attr-char     = ALPHA / DIGIT
+                   / "!" / "#" / "$" / "&" / "+" / "-" / "."
+                   / "^" / "_" / "`" / "|" / "~"
+                   ; token except ( "*" / "'" / "%" )
+    *
+    * egHeader: attachment;filename*=utf-8''%E4%BD%A0%E5%A5%BD.txt
+    * Reference: https://github.com/jshttp/content-disposition
+    */
+    // eslint-disable-next-line max-len
+    encodedFileNameRegex = /;[ \t]*(?:filename\*)[ \t]*=[ \t]*([A-Za-z0-9!#$%&+\-^_`{}~]+)'.*'((?:%[0-9A-Fa-f]{2}|[A-Za-z0-9!#$&+.^_`|~-])+)[ \t]*/,
+
+    /**
+    * @private
+    * RegExp to match quoted-pair in RFC 2616
+    *
+    * quoted-pair = "\" CHAR
+    * CHAR        = <any US-ASCII character (octets 0 - 127)>
+    */
+    quotedPairRegex = /\\([ -~])/g,
+
+    /**
+     * @private
+     * Regex to match all the hexadecimal number inside encoded string
+     */
+    hexCharMatchRegex = /%([0-9A-Fa-f]{2})/g,
+
+    /**
+     * @private
+     * Regex to match non-latin characters
+     */
+    nonLatinCharMatchRegex = /[^\x20-\x7e\xa0-\xff]/g,
+
+    /**
+     * Replaces non-latin characters with '?'
+     *
+     * @private
+     * @param {String} val - The string which needs to be checked and replaced
+     */
+    getLatin = function (val) {
+        return val.replace(nonLatinCharMatchRegex, TOKEN_QUESTION_MARK);
+    },
+
+    /**
+     * Decodes the hexcode to charCode
+     *
+     * @private
+     * @param {String} str - The matched string part of a hexadecimal number
+     * @param {String} hex - The hexadecimal string which needs to be converted to charCode
+     * @returns {String} String with decoded hexcode values
+     */
+    decodeHexcode = function (str, hex) {
+        return String.fromCharCode(parseInt(hex, Number_16));
+    },
+
+    /**
+     * HashMap for decoding string with supported characterSets
+     * iso-8859-1
+     * utf-8
+     * @private
+     */
+    characterDecoders = {
+        'iso-8859-1': getLatin,
```

Comment:
Why not define it here directly? `getLatin` I don't see `getLatin` being used anywhere directly.